src/iosched/unified.c

```diff
@@ -450,22 +496,26 @@ int unified_close(struct dentry *d, bool flush, void *iosched_handle)
 {
 	int write_error, ret = 0;
 	struct unified_data *priv = iosched_handle;
+	struct dentry_priv *dpr;
```

Comment:
It's good idea to initialize this one to NULL, just in case

Reply:
Fixed in my branch.